os/board/rtl8721csm/src/component/os/tizenrt/lwip_intf_tizenrt.c

```diff
@@ -231,11 +232,13 @@ unsigned char *netif_get_hwaddr(int idx_wlan)
 {
 	struct netdev *dev_tmp = NULL;
 	unsigned char *dev_addr;
+	dev_addr = (unsigned char *)malloc(sizeof(unsigned char) * (IFHWADDRLEN));
```

Comment:
`malloc` is allocating the memory into the user heap when binary separation.
Is it okay to allocate the memory into the user heap?
If not, please use `kmm_malloc` instead of `malloc`.

Reply:
Please see my next comment